.gitignore

```diff
@@ -9,3 +9,4 @@
 /tmp/
 
 *.gem
+.ruby-version
```

Comment:
Hello @bruce-fs -- This change looks unrelated to the current PR. Can you remove this? Thanks.